package.json

```diff
@@ -12,7 +12,9 @@
     "@types/react-dom": "^18.0.0",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
+    "react-router-dom": "^6.14.2",
     "react-scripts": "5.0.1",
+    "styled-components": "^6.0.6",
     "typescript": "^4.4.2",
```

Comment:
I see the typescript version is a pretty old one. Was it generated like this by CRA? Or you set it to this version for a particular reason. No need to change it, I just want to understand :)

Reply:
Yes, this was how it was generated by CRA, on my laptop the version that is generally installed is 5.0.3.